test/Npgsql.Tests/SystemTransactionTests.cs

```diff
@@ -44,14 +44,16 @@ public void ExplicitEnlist()
                 using (var scope = new TransactionScope())
                 {
                     conn.EnlistTransaction(Transaction.Current);
-                    Assert.That(conn.ExecuteNonQuery(@"INSERT INTO data (name) VALUES ('test')"), Is.EqualTo(1));
+                    Assert.That(conn.ExecuteNonQuery(@"INSERT INTO data (name) VALUES ('test')"), Is.EqualTo(1),
+                        "Unexpected insert rowcount");
```

Comment:
I have added messages in a bunch of cases where the Assert failure could be ambiguous with another assert in the same test.